pubspec.yaml

```diff
@@ -9,10 +9,12 @@ environment:
 
 dependencies:
   meta: ^1.0.4
+  transformer_utils: ^0.1.1
```

Comment:
Will this become a dependency in all consumers? (Is it already an implicit dependency in all consumers?)